website/project/views/node.py

```diff
@@ -140,7 +140,7 @@ def project_new_post(auth, **kwargs):
 
     else:
         try:
-            project = new_node(category, title, user, description)
+            project = new_node(category, title, user, description, campaign=campaign)
```

Comment:
Nodes created through v2 endpoints doesn't need system tags added to them because they are not campaign-related. However when we move the creation of campaign-related nodes from V1 to V2 api, we will need to move this over. For now, I'll just add a TODO for posterity.

Reply:
I wonder if this would be something good to document in @brianjgeiger's requirements - seems easy to forget to do this